### PR TITLE
Some GCC 6 fixes

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

**Review by xen0l — APPROVED** [review]

File: components/library/libgnomeui/patches/libgnomeui-04-gcc6.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+--- libgnomeui-2.24.5/libgnomeui/gnome-dateedit.c.orig	Fri Apr 28 19:30:02 2017
```
Comment (xen0l) [inline components/library/libgnomeui/patches/libgnomeui-04-gcc6.patch]: Patch is added, so we need to bump component revision

File: components/library/wxwidgets/patches/01-gcc6.patch [hunk 2]
```diff
@@ -0,0 +1,107 @@
+
```
Comment (xen0l) [inline components/library/wxwidgets/patches/01-gcc6.patch:1]: Bump component_revision, so patch is applied.

**Review by xen0l — CHANGES_REQUESTED** [review]

File: components/library/wxwidgets/patches/01-gcc6.patch (hunk 2 above)
Comment (alarcher) [reply components/library/wxwidgets/patches/01-gcc6.patch:1]: Done.

File: components/library/libgnomeui/patches/libgnomeui-04-gcc6.patch (hunk 1 above)
Comment (alarcher) [reply components/library/libgnomeui/patches/libgnomeui-04-gcc6.patch]: Done.

File: components/library/libgnomeui/Makefile [hunk 3]
```diff
@@ -13,12 +13,12 @@
 # Copyright 2014 Alexander Pyhalov.  All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
```
Comment (pyhalov) [inline components/library/libgnomeui/Makefile]: perhaps it's better to move gnome2 libraries to desktop/gnome2/ ?

Comment (alarcher) [reply components/library/libgnomeui/Makefile]: I initially put them in desktop/gnome2 but the category layout specifies library/* to go to library. What do you prefer?

Comment (pyhalov) [reply components/library/libgnomeui/Makefile]: do it the same way with mate libs

**Comment by alarcher** [comment]
Updated.

**Pull request merged**